tool/travis.sh

```diff
@@ -94,5 +94,10 @@ else
     coveralls-lcov var/lcov.info
 
     exit $status
+  else
+    # Run the tests w/o coverage
+    dart --disable-service-auth-codes \
```

Comment:
We may not need `--disable-service-auth-codes` (but I don't think having it hurts anything).

Reply:
Ah, right.  Good catch.  I'll remove.  Thanks!